### PR TITLE
chore(l10n): [zh_Hans] update locale

### PR DESCRIPTION
**Pull request by lolosiax**
The lonelyicer changes were merged. It is recommended to merge lonelyicer pull requests first.
https://github.com/vrc-get/vrc-get/pull/1567

### CODE REVIEW

File: vrc-get-gui/locales/zh_hans.json5 [hunk 1]
```diff
@@ -95,12 +95,18 @@
...
+    "projects:last modified:hours_one": "{{count}} 小时前",
+    "projects:last modified:hours_other": "{{count}} 小时前",
+    "projects:last modified:days_one": "{{count}} 天前",
+    "projects:last modified:days_other": "{{count}} 天前",
+    "projects:last modified:weeks_one": "{{count}} 周前",
+    "projects:last modified:weeks_other": "{{count}} 周前",
+    "projects:last modified:months_one": "{{count}} 月前",
+    "projects:last modified:months_other": "{{count}} 月前",
+    "projects:last modified:years_one": "{{count}} 年前",
+    "projects:last modified:years_other": "{{count}} 年前",
```
Comment (anatawa12) [inline vrc-get-gui/locales/zh_hans.json5:109]: It looks _one and _other are same so `projects:last modified:minutes` is enough.

_one and _other is for some languages that have different localization is needed, like English need 1 minute and 2 minute**s**

(Non blocking)

**Comment by anatawa12** [comment]
@lonelyicer, could you check the localization?

File: vrc-get-gui/locales/zh_hans.json5 (hunk 1 above)
Comment (lolosiax) [reply vrc-get-gui/locales/zh_hans.json5:109]: > _one and _other is for some languages that have different localization is needed, like English need 1 minute and 2 minutes  

Inconsistencies in line numbers can cause difficulties in translation, is it feasible to use comments to fill in the blank lines caused by discrepancies?

**Comment by lolosiax** [comment]
I am checking the `vpm repositories:tooltip:remove curated or official repository` modified a few minutes ago. There is some confusion in this place. I will update again later.

File: vrc-get-gui/locales/zh_hans.json5 (hunk 1 above)
Comment (anatawa12) [reply vrc-get-gui/locales/zh_hans.json5:109]: Inconsistencies in line numbers can cause difficulties in translation, is it feasible to use comments to fill in the blank lines caused by discrepancies?

Currently @lonelyicer is the maintainer of the Simplified Chinese so I want to ask lonelyicer for decisition.
Possible options are:
- keep line numbers with blank line
- keep line numbers with comment line
- keep line numbers with duplicating keys with _other and _one
- do not duplicate keys and allow line number difference

Comment (lolosiax) [reply vrc-get-gui/locales/zh_hans.json5:109]: > Currently @lonelyicer is the maintainer of the Simplified Chinese.

Ok, thanks for reminding me that I was mistaken for a regular contributor. The maintainer's opinion is important.

File: vrc-get-gui/locales/zh_hans.json5 [hunk 2]
```diff
@@ -130,6 +136,12 @@
     "projects:dialog:select unity header": "选择 Unity",
     "projects:dialog:multiple unity found": "发现该版本有多个 Unity 安装。",
     "projects:dialog:keep using this version": "继续使用此版本",
+    "projects:dialog:unity not found but china found": "Unity 不存在，但检测到对应的中国版",
```
Comment (lonelyicer) [inline vrc-get-gui/locales/zh_hans.json5:139]: `"未找到匹配的 Unity 版本，但找到了对应的中国版 Unity。"`

File: vrc-get-gui/locales/zh_hans.json5 [hunk 3]
```diff
@@ -130,6 +136,12 @@
...
+    "projects:dialog:unity version of the project not found but china found": "无法找到该项目使用的 Unity {{expectedUnity}} 。<br>但是，我们在你的设备中检测到对应的中国版 Unity {{chinaUnity}}。<br>要为这个项目使用中国版 Unity 吗？",
```
Comment (lonelyicer) [inline vrc-get-gui/locales/zh_hans.json5:140]: `"无法找到该项目使用的 Unity {{expectedUnity}} 。<br>但是，我们在您的设备中找到了对应的中国版 Unity {{chinaUnity}}。<br>要让这个项目使用中国版 Unity 吗？"`

File: vrc-get-gui/locales/zh_hans.json5 [hunk 4]
```diff
@@ -130,6 +136,12 @@
...
+    "projects:dialog:use china version": "使用中国版 Unity",
+    "projects:dialog:unity not found but international found": "Unity 不存在，但检测到对应的国际版",
```
Comment (lonelyicer) [inline vrc-get-gui/locales/zh_hans.json5:142]: `"未找到匹配的 Unity 版本，但找到了对应的国际版 Unity。"`

File: vrc-get-gui/locales/zh_hans.json5 [hunk 5]
```diff
@@ -130,6 +136,12 @@
...
+    "projects:dialog:unity version of the project not found but international found": "无法找到该项目使用的 Unity {{expectedUnity}} 。<br>但是，我们在你的设备中检测到对应的国际版 Unity {{internationalUnity}}。<br>要为这个项目使用国际版 Unity 吗？",
```
Comment (lonelyicer) [inline vrc-get-gui/locales/zh_hans.json5:143]: `"无法找到该项目使用的 Unity {{expectedUnity}} 。<br>但是，我们在你的设备中找到了对应的国际版 Unity {{internationalUnity}}。<br>要让这个项目使用国际版 Unity 吗？"`

File: vrc-get-gui/locales/zh_hans.json5 [hunk 6]
```diff
@@ -174,6 +188,7 @@
 
     "projects:manage:suggest unity migration": "您的项目使用的是 VRChat SDK 不再支持的 Unity 2019。VRChat 建议您将项目迁移到 Unity 2022。",
     "projects:manage:suggest unity patch migration": "您的项目使用的是较旧版本的 Unity 2022。VRChat 建议您将项目迁移到较新版本的 Unity 2022。",
+    "projects:manage:suggest unity china to international migration": "您的项目使用的是中国版 Unity。建议您将项目迁移到国际版 Unity，以便更好地兼容 VRChat。",
```
Comment (lonelyicer) [inline vrc-get-gui/locales/zh_hans.json5:191]: `"您的项目使用的是中国版 Unity。我们建议您将项目迁移到国际版 Unity，以便更好地兼容 VRChat。"`

File: vrc-get-gui/locales/zh_hans.json5 [hunk 7]
```diff
@@ -220,6 +247,7 @@
     "projects:manage:dialog:upgrade major": "您将要把项目使用的 Unity 版本升级到另一个主要版本。这将会变更您的库及脚本且需要重新构建。<br/>请在继续前备份您的项目。<br/>您想要继续吗？",
     "projects:manage:dialog:upgrade major vrchat supported": "您将要迁移项目的 Unity 版本。同时将会升级 VRCSDK，这可能会变更您的库及脚本且需要重新构建。<br/>请在继续前备份您的项目。<br/>您想要继续吗？",
     "projects:manage:dialog:upgrade major vrchat unsupported": "您将要迁移项目的 Unity 版本。同时将会升级 VRCSDK，这可能会变更您的库及脚本且需要重新构建。<br/>请在继续前备份您的项目。<br/>此外，该版本是一个不被 VRChat 支持的版本，所以在升级后您将无法继续上传您的项目。<br/>您想要继续吗？",
+    "projects:manage:dialog:changing china releases": "您将要在中国版与国际版之间改变 Unity 版本。这可能会变更您的库及脚本且需要重新构建。<br/>请在继续前备份您的项目。<br/>您想要继续吗？",
```
Comment (lonelyicer) [inline vrc-get-gui/locales/zh_hans.json5:250]: `"您将要迁移项目的 Unity 版本到不同地区的版本。这可能会变更您的库及脚本且需要重新构建。<br/>请在继续前备份您的项目。<br/>您想要继续吗？"`

**Review by lonelyicer — CHANGES_REQUESTED** [review]

File: vrc-get-gui/locales/zh_hans.json5 (hunk 1 above)
Comment (lonelyicer) [reply vrc-get-gui/locales/zh_hans.json5]: `Keeping line numbers with blank lines` may break the file classification structure.
`Keeping line numbers with comment lines` works, but makes the file look a bit cluttered.
`keep line numbers with duplicating keys with _other and _one` keeps non-essential keys, which can be a little tricky to maintain.
`do not duplicate keys and allow line number difference` may indeed cause some difficulty, but much less for the difficulties mentioned above.

So my suggestion is `do not duplicate keys and allow line number differences`, but `keep line numbers with comment lines` is also acceptable.

**Review by lonelyicer — APPROVED** [review]

**Comment by anatawa12** [comment]
I cannot read Chinese so I'm asking for maintainer for checking your PR.

**Review by lonelyicer — APPROVED** [review]

**Comment by lolosiax** [comment]
> I cannot read Chinese so I'm asking for maintainer for checking your PR.

Sorry, I asked lonelyicer first, and he agreed an hour ago.

**Pull request merged**